hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/TopicsEndpoint.java

```diff
@@ -4,18 +4,16 @@
 import com.wordnik.swagger.annotations.ApiOperation;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
-import pl.allegro.tech.hermes.api.PatchData;
-import pl.allegro.tech.hermes.api.Query;
-import pl.allegro.tech.hermes.api.Topic;
-import pl.allegro.tech.hermes.api.TopicMetrics;
-import pl.allegro.tech.hermes.api.TopicName;
+import pl.allegro.tech.hermes.api.*;
```

Comment:
\[Checkstyle\] ERROR: Using the '\.\*' form of import should be avoided \- pl\.allegro\.tech\.hermes\.api\.\*\.